src/contracts/Qearn.h

```diff
@@ -0,0 +1,612 @@
+using namespace QPI;
+
+#define MINIMUM_LOCKING_AMOUNT 10000000
```

Comment:
all `#define` is prohibited. Please change these `#define` to `constexpr unsigned long long`